src/components/DateTemperature/MainTemperatureBlock/index.js

```diff
@@ -0,0 +1,50 @@
+import dayjs from "dayjs";
+import { useEffect, useState } from "react";
+import { getTemperature } from "../../../utilities/helpers/get-temperature";
+import "./MainTemperatureBlock.scss";
+
+export const MainTemperatureBlock = ({ dayTemperatureList, currentDate }) => {
+	const [time, setCurrentTime] = useState("");
+	useEffect(() => {
+		setInterval(() => setCurrentTime(new Date()), 1000);
+	}, []);
+	const arrayOfWeatherParameters = [{ title: '', data: dayTemperatureList.map(item => item.time.split(" ").splice(1, 1).join(' ')), class: "app__time" },
```

Comment:
separate

Reply:
do u need to initialize the array const for each rerender every one second? use a hook useMemo

Reply:
rename the variable to tableData

Reply:
where is the fixed code?